src/main/resources/templates/db_scripts/schema.json

```diff
@@ -61,12 +68,11 @@
     {
     "run": "after",
     "snippetPath": "update_started_date_before_insertion.sql",
-    "fromModuleVersion": "1.0"
+    "fromModuleVersion": "mod-kb-ebsco-java-3.0.1"
     },
     {
       "run": "after",
-      "snippetPath": "drop_set_id_in_jsonb.sql",
-      "fromModuleVersion": "mod-kb-ebsco-java-3.0.2"
```

Comment:
drop_set_id_in_jsonb.sql shouldn't have fromModuleVersion, because it has to run even if it wasn't changed (set_id_in_jsonb trigger can be added when running scripts for other tables, so we always need to run drop_set_id_in_jsonb.sql after all other scripts).